capture_service/dive_client_cli.cc

```diff
@@ -113,6 +121,27 @@ download_path,
 ".",
 "specify the full path to download the capture on the host, default to current directory.");
 
+ABSL_FLAG(
+std::string,
+device_architecture,
+"x86",
+"specify the device architecture to capture with gfxr. If not specified, the default is x86.");
+ABSL_FLAG(std::string,
+          gfxr_capture_file_dir,
+          "gfxr_capture",
+          "specify the name of the directory for the gfxr capture. If not specified, the default "
+          "file name is gfxr_capture.");
+ABSL_FLAG(int,
+          frame,
+          -1,
+          "specify which frame to capture with gfxr. If not specified, the default is -1.");
+ABSL_FLAG(
+std::string,
+frame_range,
+"",
+"specify the range of frames to capture with gfxr. If not specified, the default is frame "
```

Comment:
a bit confused here, if the user does not specify frame_range, what is the default value?

Reply:
Default will just be an empty string which is checked in the main() function

Reply:
i think we could add some descriptions of the behavior of the case when the default value is used.

Reply:
the goal here is to provide more info about the default behavior so that the user knows what to expect :)

Reply:
Ok, added more of a description about the default values for each of the capture frame flags.

Reply:
thanks